android/assets/jsons/translations/German.properties

```diff
@@ -3107,8 +3115,7 @@ Moai = Moai
 Cannot be built on bonus resource = Darf nicht auf Bonus-Ressourcen gebaut werden
 Terrace farm = Terrassenfelder
 
- # Requires translation!
-Unpillagable = 
+Unpillagable = Darf nicht geplündert werden
```

Comment:
Kenne den Kontext nicht aber wäre "Kann nicht geplündert werden" passender?

Reply:
Kontext = markiert spezielle tile 'improvements' - alte Ruinen, Barbaren-camps und vor allem Stadtzentren die konnten nämlich geplündert werden noch vor kurzem, und so konnte man tatsächlich ein Stadtzentrum incl. Grafik außerhalb einer Stadt er-tricksen... War vorher hardcoded für alle anderen und city center vergessen worden, jetzt im json und damit in civilopedia sichtbar.

Formulierungen beide gleich OK. Hauptsache es ist was nicht irreführendes drin. Lass es so reinnehmen, und wenn bei Deiner nächsten Runde Du immer noch 'Kann' besser findest, bau's rein.

Vielleicht sollte ich ein "feature" bauen das alternative Texte erlaubt und dann nach Wochentag oder sonstigen langsam wechselnden Einflüssen einen auswählt - und zu Ostern gibt's dann "Finger weg! Hier wird nicht geplündert"...